### PR TITLE
TASK: load page after creation

### PR DESCRIPTION
**Pull request by dimaip**

### CODE REVIEW

File: Classes/Neos/Neos/Ui/Domain/Model/AbstractChange.php [hunk 1]
```diff
@@ -95,13 +95,13 @@ protected function reloadDocument()
...
-     *
+     * @param NodeInterface $subject
      * @return void
      */
-    protected function addDocumentNodeCreatedFeedback()
+    protected function addDocumentNodeCreatedFeedback($subject = null)
     {
+        $node = $subject ?: $this->getSubject();
         $nodeService = new NodeService();
-        $node = $nodeService->getClosestDocument($this->getSubject());
```
Comment (dimaip) [inline Classes/Neos/Neos/Ui/Domain/Model/AbstractChange.php]: @grebaldi why was this `getClosestDocument` necessary? The method is called `addDocumentNodeCreatedFeedback`, so I assume it should only create feedback for document nodes, no?

**Comment by dimaip** [comment]
Fixes #418

**Comment by dimaip** [comment]
Fixes #407

File: Classes/Neos/Neos/Ui/Domain/Model/AbstractChange.php (hunk 1 above)
Comment (skurfuerst) [inline Classes/Neos/Neos/Ui/Domain/Model/AbstractChange.php]: correct!

**Pull request merged**